lib/scrabble.rb

```diff
@@ -1,7 +1,33 @@
+require "pry"
 class Scrabble
 
+  attr_reader :word
+
+  def initialize
+    @word = word
+  end
+
+  def score
+    letters = word.split('')
```

Comment:
Need to `#upcase` if we're going to get a result out of the `point_values` hash.